pace_neutrons_cli/utils.py

```diff
@@ -131,7 +131,7 @@ def CachedCTFs(self):
             retval = self.config['pace']['CachedCTFs'].split(';')
         except KeyError:
             retval = []
-        return [r for r in retval if r != '']
+        return list(filter(None, retval))
```

Comment:
As I mentioned I wouldn't actually use this form. It's confusing to those who don't know what it does
```suggestion
        return [val for val in retval if val]
```